documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc

```diff
@@ -0,0 +1,28 @@
+= Fuse Tooling - What's New in 10.3
+:page-layout: whatsnew
+:page-component_id: fusetools
+:page-component_version: 10.3.0.AM2
+:page-product_id: jbt_core
+:page-product_version: 4.5.3.AM2
+
+== Display Fuse version corresponding to Camel version proposed
+
+When creating a new project, the Camel version can be selected among a long list. Now, the Fuse version is also displayed to help choosing version corresponding to the production version that users are using/targeting.
+
+image::./images/listOfFuseVersion.png[Fuse Version also displayed in drop-down list close to Camel version]
+
+== Update validation for similar ids between component and its definition
+
+Since Camel 2.20, it is now possible to use similar ids between the component name and its definition unless the specific property "registerEndpointIdsFromRoute" is provided.
```

Comment:
Starting with Camel 2.20, you can use similar IDs for the component name and its definition ...